### PR TITLE
Feature/hero-video

### PR DESCRIPTION
**Pull request by Naraim**
# FooCamp - TransformaSion

## Trello ticket
[Hero overlay](https://trello.com/c/DnjwHzfT/17-5-homepage-hero-video-overlay)

## Description
Added modal component

## To reproduce
1. Run the project
2. Open the specific page (e.g. http://localhost:8080/marian-page.html)
3. Check the component

## Screenshots
image
image

### CODE REVIEW

**Review by jortizhuge — APPROVED** [review]

File: src/scss/components/_modal.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.modal{
+
+    &__close-btn{
```
Comment (jsebax) [inline src/scss/components/_modal.scss:3]: Please add a `cursor: pointer;` property to the close button

File: src/js/modal.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+function openModal(event) {
+    const target = event.target;
+    const modal = document.querySelector(`#${target.dataset.open}`);
+    const body = document.querySelector('body');
```
Comment (jsebax) [inline src/js/modal.js]: This can be accessed like `document.body`.

File: templates/hero.hbs [hunk 3]
```diff
@@ -15,4 +15,14 @@
...
       </div>
    </div>
+   {{!-- Modal --}}
+   <div class="modal fade" id="modal-video-hero" tabindex="-1">
+      <div class="modal-dialog modal-dialog-centered">
+         <button type="button" class="modal__close-btn"><i class="icon fas fa-times"></i></button>
+         <video class="modal__video" src="{{assetSrc video}}" controls ></video>
+         {{!-- <div class="modal-container">
+            
+         </div> --}}
```
Comment (jsebax) [inline templates/hero.hbs]: If you don't need this anymore, please remove the commented code

**Comment by jsebax** [comment]
Please solve the conflicts

**Review by jsebax — APPROVED** [review]

**Pull request merged**